docs/examples/AsyncCallbacks.tsx

```diff
@@ -22,24 +18,6 @@ const loadOptions = (
   }, 1000);
 };
 
-export default class WithCallbacks extends Component<{}, State> {
-  state: State = { inputValue: '' };
```

Comment:
I've removed the state logic to simplify the example. I don't think it was necessary to demonstrate how to use the async callbacks api.